script.skinshortcuts/addon.xml

```diff
@@ -37,4 +38,8 @@
 		<email></email>
 		<source>https://github.com/BigNoid/script.skinshortcuts</source>
 	</extension>
+	<assets>
+		<icon>icon.png</icon>
+		<fanart>fanart.jpg</fanart>
+	</assets>
```

Comment:
Cheers Ronie. I've moved it to the xbmc.addon.metadata for 1.0.12 - https://github.com/xbmc/repo-scripts/pull/306. If you could confirm it's now in the right place that would be great.